mbl-core/tests/partitions/test_partitions.py

```diff
@@ -128,3 +173,13 @@ def get_actual_part_table():
 
 def test_part_table():
     assert get_actual_part_table() == get_expected_part_table()
+
+
+def test_part_device_name_files():
+    device_name = get_block_device_for_parts()
```

Comment:
device_name is "mmcblk1" or similar. Is DEVICE_NAME_BANK the actual partition or the parent ?

Reply:
`MBL_<part-name>_DEVICE_NAME_BANK<bank-number>` holds the parent device name (`PKNAME` in `lsblk` terminology; `mmcblk1` or similar) for the `<bank-number>` bank of the `<part-name>` partition.